document/src/main/java/com/ritense/document/service/impl/JsonSchemaDocumentService.java

```diff
@@ -582,7 +582,7 @@ public void assignUserToDocument(UUID documentId, String assigneeId) {
                 new DelegateUserEntityAuthorizationRequest<>(
                     JsonSchemaDocument.class,
                     ASSIGNABLE,
-                    assignee.getEmail(),
```

Comment:
@theo-ritense 
> We're going back to assigning by user id?

We're in the process of switching from email to user-id. See ticket: #68171